lib/csr_matrix.rb

```diff
@@ -13,7 +13,7 @@ def initialize(a_array, ia_array, ja_array, row_count, column_count)
     @column_count = column_count
   end
 
-  def self.rows(rows, column_count = rows[0].length)
+  def self.rows(rows, row_count = rows.length, column_count = rows[0].length)
```

Comment:
We didn't specify row_count as an argument because we were making it similar to Matrix.rows(rows, column_count = rows[0].length)

Reply:
oh okay ill fix that